apps/backend/src/sales/sales.service.ts

```diff
@@ -38,4 +38,58 @@ export class SalesService {
       throw e;
     }
   }
+
+  async queryCashTransactions(
```

Comment:
Unsure about the function of this query in this service, please clarify.

Reply:
This should return all of the payments for Cash/Cheque (to match against TDI17). 

I am trying to query by the current and last deposit dates. This will probably need to be changed. I have not yet found a rule for date range that works for each location. 

We should add a `and match=false`. Maybe then we can query for anything less than the current deposit date, as all of the previous payments should be accounted for? (and if they are not then an exception could be thrown?)